experiment.py

```diff
@@ -201,101 +233,146 @@ def get_samples_around_wildtype(
     return sample_df
 
 
+def get_pdb_from_mogwai_filepath(mogwai_filepath):
+    pdb = mogwai_filepath.rsplit('/', 1)[1].rstrip('_model_state_dict.npz')
+    return pdb
+
+
+def get_test_sets_in_dir(directory: PathLike) -> Dict[str, np.ndarray]:
+    """Returns a mapping from test set names to an array of integer encoded sequences.
+
+    Loads all of the .npz files in the given directory. Assumes that the npz
+    files have an attribute named 'sequences' which can be accessed.
+
+    Args:
+        directory: The directory to load from.
+
+    Returns:
+        A mapping from test set names to an array of integer encoded sequences.
```

Comment:
not necessary to change, but I find it a bit awkward that the test set file only contains the sequences, not the labels. This seems vulnerable to bugs where the sequences are selected wrt a different landscape (e.g. different tuning values) than what is used for doing the actual regression experiment.

Reply:
I think this is good to keep in mind. We happen to be in a regime where the way we select sequences doesn't depend on the tuning. The top epistatic pairs will continue to be the top epistatic pairs, and the top singles will continue to be the top singles, even after arbitrary scaling and shifting.

Reply:
can you add a code comment summarizing your response here?

Reply:
Yes. I put it in the docstring of `get_test_sets`